src/ts/order.ts

```diff
@@ -126,6 +131,32 @@ export function timestamp(t: Timestamp): number {
   return typeof t === "number" ? t : ~~(t.getTime() / 1000);
 }
 
+/**
+ * Normalizes an app data value to a 32-byte hash.
+ * @param hashLike A hash-like value to normalize.
+ * @returns A 32-byte hash encoded as a hex-string.
+ */
+export function hashify(h: HashLike): string {
+  return typeof h === "number"
+    ? `0x${h.toString(16).padStart(64, "0")}`
```

Comment:
I learned from you to love Ethers utils. :smile: 
Why not using them here? It also checks for overflows.
```suggestion
    ? ethers.utils.defaultAbiCoder.encode(["uint256"], [h])
```

Reply:
I didn't even think about it! Much better :smile: